official/recommendation/data_async_generation.py

```diff
@@ -157,75 +176,80 @@ def init_worker():
   signal.signal(signal.SIGINT, sigint_handler)
 
 
-def _construct_training_records(
+def _construct_records(
```

Comment:
Won't padding the final training batch every epoch slow down training? Why remove spillover?

Reply:
Rules. Given that this thing trains at ~100 batches/sec, it is unlikely to cause too much harm.